meta-resin-common/recipes-containers/docker/docker/docker.service

```diff
@@ -9,10 +9,10 @@ Before=dnsmasq.service
 [Service]
 Type=notify
 Restart=always
-ExecStart=/usr/bin/dockerd --log-driver=journald -s @DOCKER_STORAGE@ -H fd:// --dns 172.17.0.1
+ExecStart=/usr/bin/dockerd --log-driver=journald -s @DOCKER_STORAGE@ -H fd:// --dns 10.114.101.1 --bip 10.114.101.1/24 --fixed-cidr=10.114.101.0/25
```

Comment:
docker.conf is still needed, it is docker.conf who actually defines the final ExecStart directive. It's a drop-in and it will overwrite the ExecStart from the service file